source/backend/api/Services/IDispositionFileService.cs

```diff
@@ -18,5 +18,9 @@ public interface IDispositionFileService
         IEnumerable<PimsDispositionFileProperty> GetProperties(long id);
 
         IEnumerable<PimsDispositionFileTeam> GetTeamMembers();
+
+        IEnumerable<PimsDispositionOffer> GetOffers(long dispositionFileId);
+
+        IEnumerable<PimsDispositionSale> GetSales(long dispositionFileId);
```

Comment:
Agree with Devin that this should return a single instance of **PimsDispositionSale** (or null)

Reply:
I am a little conflicted with this because the Entity Classes from the Scaffolding do not declare the relationship as such.